packages/lodestar-cli/src/cmds/beacon/cmds/init/init.ts

```diff
@@ -1,14 +1,36 @@
+import path from "path";
 import {Arguments} from "yargs";
+import deepmerge from "deepmerge";
 
+import {rootDir} from "../../../../options";
+import {beaconDir} from "../../options/beaconDir";
 import {IBeaconArgs} from "../../options";
 import {mkdir} from "../../../../util";
 import {initPeerId, initEnr, readPeerId} from "../../../../network";
 import {initBeaconConfig} from "../../config";
+import {getTestnetConfig, downloadGenesisFile, fetchBootnodes} from "../../testnets";
 
 /**
  * Initialize lodestar-cli with an on-disk configuration
  */
 export async function init(args: Arguments<IBeaconArgs>): Promise<void> {
+  // Auto-setup altona
+  if (args.testnet) {
+    const altonaConfig = getTestnetConfig(args.testnet);
```

Comment:
should these variable names reference altona now? we'll likely support "the multiclient testnet", and other testnets, as they come out